src/api/.rubocop.yml

```diff
@@ -43,11 +43,15 @@ Style/AlignParameters:
 # Configuration parameters: AllowSafeAssignment.
 Lint/AssignmentInCondition:
   Enabled: true
-
-# Henne
```

Comment:
Quite a while ago we picked some rubocop rules we wanted to enable, and fix the code using those old rules. Thus we assigned some of the rules by adding such name tags. In the end not everyone did it;-) So this is just a relict you can ignore.